licenses/VXL

```diff
@@ -0,0 +1,30 @@
+Copyright 2000-2013 VXL Contributors
```

Comment:
That license looks like it is already present in the tree. Please use on of those instead.